test/gurka/gurka.cc

```diff
@@ -52,16 +52,6 @@ std::vector<midgard::PointLL> to_lls(const nodelayout& nodes,
   return lls;
 }
 
-/**
- * build a valhalla json request body
- *
- * @param location_type  locations or shape
- * @param waypoints      sequence of pointlls representing the locations
- * @param costing        which costing name to use, defaults to auto
- * @param options        overrides parts of the request, supports rapidjson pointer semantics
- * @param stop_type      break, through, via, break_through
- * @return json string
- */
```

Comment:
this was an accident, but we could leave it this way. the header has the same description, no point in duplicating docstrings

Reply:
.. forget it.. the whole "problem" is that I switched the definition to the header.. i'll revert while fixing the test bug failing CI